asciidoctorj-api/src/main/java/org/asciidoctor/ast/Author.java

```diff
@@ -0,0 +1,28 @@
+package org.asciidoctor.ast;
+
+public interface Author {
+
+    public String getFullName();
+
+    public void setFullName(String fullName);
+
+    public String getLastName();
+
+    public void setLastName(String lastName);
+
+    public String getFirstName();
+
+    public void setFirstName(String firstName);
+
+    public String getMiddleName();
+
+    public void setMiddleName(String middleName);
+
+    public String getEmail();
+
+    public void setEmail(String email);
+
+    public String getInitials();
+
+    public void setInitials(String initials);
```

Comment:
I think all these setters should go be on the impls only.
This API is only to extract structural information about the document, there's no way to modify these objects and have an impact on the underlying document.